package.json

```diff
@@ -76,6 +76,7 @@
     "blue-tape": "^1.0.0",
     "bundlesize": "^0.12.0",
     "contentful-sdk-jsdoc": "^2.2.0",
+    "core-js": "^2.5.1",
```

Comment:
why bringing back `core-js`

Reply:
Because it is a dependency of our code which was missing in one build I did. We should not rely on sub-dependencies. (It was babel in that case)

Reply:
https://github.com/contentful/contentful.js/blob/next/webpack.config.js#L90-L96